src/static/css/cards.css

```diff
@@ -0,0 +1,37 @@
+.card-image{
+  display:block;
+  max-width:35% !important;
```

Comment:
In this case the id approach won't work I think. Because the classes with the !important are used to style at least 2 at most 5 elements with the same class, so it would be cumbersome and ugly to add different id-s for the parent elements, and then target them as their children.